src/Symfony/Component/Form/Form.php

```diff
@@ -748,7 +748,7 @@ public function isEmpty()
 
         return FormUtil::isEmpty($this->modelData) ||
             // arrays, countables
-            0 === count($this->modelData) ||
+            ((is_array($this->modelData) || $this->modelData instanceof \Countable) && 0 === count($this->modelData)) ||
```

Comment:
creating an empty array to test if some var is also an empty array seems a bit much for syntax sugar?